app/views/categories/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<p style="color: green"><%= notice %></p>
+<div >
+  <div class="login-nav">
+      <div><%= link_to "BACK", users_path, class: "rails-link" %> </div>
+      <div>CATEGORY</div>  
+      <div><%= button_to "LOG-OUT", destroy_user_session_path, method: :delete, class: "rails-btn" %></div>          
+  </div>
+
+  <% if @categories.empty? %>
+    <h2>There are no categories available</h2>
+  <% end %>
+  <% @categories.each do |category| %>
+    <%= link_to category_operations_path(category_id: category.id) do %>
+    <div class="category">
+
+      <div class="photo">
+        <% if category.icon.attached? %>
+          <%= image_tag category.icon, class: "profile_img" %>
+        <% end %>
+      </div>
+
+      <div>
+        <p><%= category.name %></p>
+        <p class="fs"><%= category.created_at.strftime("%d %b %Y") %></p>
+      </div>
+
+      <p><strong><%= category.operations.sum(:amount) %>$ </strong></p>
+
+    </div>
+      <% end %>
+  <% end %>
+
+  <div class = "footer_container">
+ <div class ="footer">
+ <p class="forgot-pwd"><%= link_to "ADD NEW CATEGORY", new_category_path(@user) %></p>
+ </div>
+ </div>
+
+  
+</div>
```

Comment:
- [ ] [REQUIRED] Kindly check the recommendation the previous reviewer dropped on the review summary where he stated that you need to stick to the Behance design color, typography: font face, size, and weight, Layout: composition, and space between elements. Kindly ensure that the nav bar, the Add new category button, and the application background has the same color as recommended in the Behance design. Kindly implement this for the rest of the pages as well to enhance uniformity.
Take a look at this:

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="308" alt="Screenshot 2023-07-19 at 22 52 49" src="https://github.com/Luchy-B/budget/assets/58692590/1f19070d-603b-4036-b42d-b470b5a76db8"> | <img width="357" alt="Screenshot 2023-07-19 at 23 02 49" src="https://github.com/Luchy-B/budget/assets/58692590/ca4f72e3-07ce-41f7-90a1-19ed139fe53a">